packages/inferno/src/DOM/unmounting.ts

```diff
@@ -28,11 +28,17 @@ export function unmount(vNode) {
 
     if (!isNull(props)) {
       const keys = Object.keys(props);
+      const doc = vNode.dom.ownerDocument || document;
```

Comment:
We could pass doc here too to avoid reading it inside loop

Reply:
doc is now just passed as argument like in other places